sphinxcontrib/mscgen.py

```diff
@@ -144,7 +145,7 @@ def render_msc(self, code, format, prefix='mscgen'):
 
     # mscgen don't support encodings very well. ISO-8859-1 seems to work best,
     # at least for PNG.
-    if isinstance(code, unicode):
+    if isinstance(code, six.text_type):
```

Comment:
I'm not convinced it is a good idea to introduce a dependency on some external library just for this check, it would make more sense to just write a helper function (it shouldn't be that hard) or simply drop Python2 support. Python2 is going out of support in less than a year anyway.

Reply:
I think it should be possible to call encode() unconditionally here. In Python2, code can be either a str or unicode, both of which support encode(). In Python3, code will always be a str here. This would eliminate the requirement for six.